### ISSUE TITLE
Lost in Guide.md

### ISSUE THREAD

**Issue by DavidBagory** [meteoric/meteor-ionic#254]
TL;DR : Can meteoric handles ionic components ?
I want to integrate meteoric in my project but I feel confuse about the framework. The guide show how to create template as well as routing with iron router but do not shows how to create ionics components. Hence I do not understand the purpose of meteoric.

For instance, If I want to display a list do I need  to :
1. make a reference of ionic in my code  `  <link href="lib/ionic/css/ionic.css" rel="stylesheet">
 <script src="lib/ionic/js/ionic.bundle.js"></script> ` 
2. write a list, as seen in the the <a href="http://ionicframework.com/docs/components/#list"> ionic documentation </a> in my index.html 
3. create a meteoric template associated with my list 
?

Or can I do this inside meteoric ?

**Comment by nickw** [comment]
From the [README](https://github.com/meteoric/meteor-ionic#build-ionic-apps-in-meteor):

> It aims to be a complete port of Ionic’s Angular directives to Meteor Blaze templates.

Ionic components are Angular directives. Meteoric attempts to replicate Ionic's Angular directives using Blaze templates.

So instead of:

```
<ion-content>
  <p>Your content here</p>
</ion-content>
```

You would write:
```
{{#ionContent}}
  <p>Your content here</p>
{{/ionContent}}
```
